frontend/.changeset/new-deers-smash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@liam-hq/erd-core": patch
+"@liam-hq/cli": patch
+---
+
+Add tooltip functionality to TableHeader component
```

Comment:
I want the release notes to help users see the value in a straightforward way.
What about the following?

```suggestion
Add tooltips to display the full table name when it is truncated in the Table node
```

Reply:
Thank you for revising the text! I think it's much better.